app/controllers/idv/cancellations_controller.rb

```diff
@@ -0,0 +1,20 @@
+module Idv
+  class CancellationsController < ApplicationController
+    include IdvSession
+
+    before_action :confirm_two_factor_authenticated
+    before_action :confirm_idv_needed
+
+    def new
+      analytics.track_event(Analytics::IDV_CANCELLATION)
```

Comment:
I realized this was not being tracked at all, so new analytics events just for you @monfresh